src/commands/generate/update_controls4delta.ts

```diff
@@ -478,6 +513,43 @@ export default class GenerateUpdateControls extends BaseCommand<typeof GenerateU
   }
 }
 
+function isXccdfFile(xccdfXmlFile: string, logger: Logger): boolean {
```

Comment:
Could this be in the utils or global so it can be reused in both the delta and update_controls4delta but only defined once?